cmd/remove_integration_test.go

```diff
@@ -512,3 +513,30 @@ func TestRemoveAllAndKeepFiles(t *testing.T) {
 		t.Errorf("directory contents (-want +got):\n%s", diff)
 	}
 }
+
+// Test removing a linked dataset after the working directory has already been deleted.
+func TestRemoveIfWorkingDirectoryIsNotFound(t *testing.T) {
+	run := NewFSITestRunner(t, "qri_test_remove_no_wd")
+	defer run.Delete()
+
+	workDir := run.CreateAndChdirToWorkDir("remove_no_wd")
+
+	// Init as a linked directory
+	run.MustExec(t, "qri init --name remove_no_wd --format csv")
+
+	// Save the new dataset
+	run.MustExec(t, "qri save")
+
+	// Go up one directory
+	parentDir := filepath.Dir(workDir)
+	os.Chdir(parentDir)
+
+	// Remove the working directory
+	err := os.RemoveAll(workDir)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Remove all should still work, even though the working directory is gone.
+	run.MustExec(t, "qri remove --revisions=all me/remove_no_wd")
```

Comment:
I'd prefer the "check" part of a test not use `Must` functions, even if just for stylistic reasons. It's not a huge thing, but a quick `if err != nil` helps the readability of what's being tested